xml_inscriptions/bib_only/MA.Bos.MFA.G.84.45.xml

```diff
@@ -62,17 +130,73 @@
       <revisionDesc>
          <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
+         <change when="2023-07-07" who="Jean Wanlass">added metadata, transcription,
+            translation</change>
       </revisionDesc>
    </teiHeader>
-   <!--facsimile element goes here-->
+   <!--  *************************************  <facsimile>  *********************************  -->
+   <!-- Used to record images of an inscription.   -->
+   
+   <facsimile>
+      <surface>
+         <desc/>
+         <graphic url="MA.Bos.MFA.G.84.45.jpg"/>
+      </surface>
+   </facsimile>
+   
    <text>
       <body>
          <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+            <ab> <gap reason="illegible" extent="unknown" unit="character"/>ε<gap reason="illegible" extent="6" unit="character"/><unclear>σ</unclear>αλ<gap reason="illegible" extent="unknown" unit="character"/> <lb/>
+               
+               <gap reason="illegible" extent="unknown" unit="character"/>ν ἐπαινέσοι<unclear>σ</unclear><supplied reason="undefined">ι</supplied> <lb/>
+               
+               <supplied reason="undefined">τοὺς ἄνδρας τοὺς ὑπὸ Τηλ</supplied>έμαχον τετα<lb break="no"></lb>
```

Comment:
if you have an `<lb break="no">`, you can still keep it as an empty element (`<lb n="x" break="no"/>', you don't need a separate close tag.